bint.go

```diff
@@ -210,48 +210,160 @@ func parseBintFromU128(s []byte) (bool, bint, uint8, error) {
 		coef u128
 		prec uint8
 	)
-	for ; pos < width; pos++ {
-		if s[pos] == '.' {
+
+	if len(s[pos:]) <= 19 {
```

Comment:
19 is a magical number here

A constant with a meaningful name, plus a comment would be better I think

Reply:
Thanks for your reviews. Yea agree this one is a little bit vauge here. 19 is the max length of a number that can be safely stored in a `uint64`. I'll change it to a constant